progenitor-impl/src/lib.rs

```diff
@@ -492,38 +501,58 @@ impl Generator {
         })
     }
 
+    fn get_dep_version(&self, name: &str) -> String {
+        match self.settings.dependency_versions.get(&name.to_string()) {
+            Some(o) => o.to_string(),
+            None => "*".to_string(),
```

Comment:
it seems like encountering a `None` here would be reflective of a bug, no?

Reply:
No. It means that `progenitor-impl` is being used without `.with_dependency_versions` being called, or not provided with an entry for one of the crates.